Project.toml

```diff
@@ -29,9 +29,9 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 VectorizationBase = "3d5dd08c-fd9d-11e8-17fa-ed2836048c2f"
 
 [compat]
-ArrayInterface = "6.0.1 - 6.0.11"
+ArrayInterface = "6"
 ArrayInterfaceCore = "0.1.5"
-ArrayInterfaceOffsetArrays = "0.1.2"
```

Comment:
Why drop the 0.1.2 requirement?

Reply:
Because the most recent changes fix the issues caused by the forwarding trait

Reply:
This was a lower bound.
Because of:
https://github.com/JuliaArrays/ArrayInterface.jl/issues/272
I cannot easily double check what actually changed in that version.

I'd be more comfortable leaving the lower bound of 0.1.2.

Reply:
Oh I didn't explicitly test the lower versions. I was just trying to get everything working with master going forward